experiments/AMIP/components/atmosphere/climaatmos_init.jl

```diff
@@ -307,22 +319,10 @@ function get_field(atmos_sim::ClimaAtmosSimulation, ::Val{:F_radiative_TOA})
         face_space = axes(atmos_sim.integrator.u.f)
         nz_faces = length(Spaces.vertical_topology(face_space).mesh.faces)
 
-        LWd_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_lw_flux_dn), face_space),
-            nz_faces - half,
-        )
-        LWu_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_lw_flux_up), face_space),
-            nz_faces - half,
-        )
-        SWd_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_sw_flux_dn), face_space),
-            nz_faces - half,
-        )
-        SWu_TOA = Fields.level(
-            CA.RRTMGPI.array2field(FT.(atmos_sim.integrator.p.radiation.radiation_model.face_sw_flux_up), face_space),
-            nz_faces - half,
-        )
+        LWd_TOA = Fields.level(CA.RRTMGPI.array2field(FT.(radiation.face_lw_flux_dn), face_space), nz_faces - half)
```

Comment:
You could in fact unpack the individual variables, without allocating, as follows:
`(; face_lw_flux_dn, face_lw_flux_up, face_sw_flux_dn, face_sw_flux_up) = atmos_sim.integrator.p.radiation.radiation_model`